### PR TITLE
added HTTP status and Xpath selection tests

### PR DESCRIPTION
**Pull request by tasmo**
no interface yet, just jmx alerts

### CODE REVIEW

File: modules/app.xql [hunk 1]
```diff
@@ -69,7 +69,8 @@ function app:get-instance($node as node(), $model as map(*), $instance as xs:str
 declare
     %templates:default("instance", "localhost")
 function app:instances($node as node(), $model as map(*), $instance as xs:string) {
-    for $current in collection($config:app-root)//instance
+    let $instances := collection($config:app-root)//instance
+    for $current in $instances[ not(@ping = "no") and not(@ping = "0") and not(@ping = "false") ]
```
Comment (joewiz) [inline modules/app.xql]: Could be simplified as `not(@ping = ("no", "0", "false"))]`

Comment (tasmo) [inline modules/app.xql]: Will do this. Thank you!

**Comment by joewiz** [comment]
Oh, you caught it already. Sorry!

**Comment by joewiz** [comment]
@tasmo Is this PR ready for review, or still a work in progress?

**Comment by adamretter** [comment]
@tasmo @wolfgangmm @joewiz  any further thoughts on this?

**Pull request closed without merging**